### PR TITLE
added E40

### LINKED ISSUE #157: Environment Improvements
### 3.1 Component model
- source code decorater is necessary
- add: user should not use other problems while Beagle runs. Best: dedicated server

### PR DESCRIPTION
**Pull request by viridispirus**
Fixes #157

### CODE REVIEW

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 1]
```diff
@@ -214,6 +214,18 @@ gls{measurement software}
...
 
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/E40/"
+
+\end_inset
+
+ The user should not run other programms while Beagle is execute.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Environment.lyx]: ... executed.
But I think "while Beagle is running" is better.

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 2]
```diff
@@ -214,6 +214,18 @@ gls{measurement software}
...
+ By reason that it falsified the measurement.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Environment.lyx]: ... falsifies...

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 3]
```diff
@@ -214,6 +214,30 @@ gls{measurement software}
...
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/E40/"
+
+\end_inset
+
+ The user should not run other programms while Beagle is running.
+ By reason that it falsifies the measurement.
```
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Environment.lyx]: "falsify" = "disprove". Use "distorts" or "influences".

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 4]
```diff
@@ -214,6 +214,30 @@ gls{measurement software}
...
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/E40/"
+
+\end_inset
+
+ The user should not run other programms while Beagle is running.
+ By reason that it distorts the measurement.
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Environment.lyx]: Maybe following is better:
"The user should not run other programs while Beagle is running as this disturbs the measurement.

Comment (viridispirus) [inline doc/Requirements Specification/Parts/Environment.lyx]: I agree, and change it.

**Comment by acBerger** [comment]
lgtm

**Comment by m1cm1c** [comment]
lgtm

**Pull request merged**